backend/database_handler/transactions_processor.py

```
@@ -914,6 +914,10 @@ def get_transactions_for_address(
         address: str,
         filter: TransactionAddressFilter,
     ) -> list[dict]:
+        try:
+            address = to_checksum_address(address)
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)